src/counter/admin.md

```diff
@@ -34,10 +34,26 @@ Let's pick [`reset`](https://raen.dev/admin/#/counter.raendev.testnet/Reset).
 
 Again you can see the comments from the code. However, now there are two new fields in the form. 
 
-- `attachedDeposit` is how much near you want to attach to the transaction; it is in yoctoNear, which is the smallest unit of NEAR (`1N = 1000000000000000000000000 yN` or `10^24`).
+- `attachedDeposit` is how much near you want to attach to the transaction; it is in [yocto](https://www.nanotech-now.com/metric-prefix-table.htm)Near, which is the smallest unit of NEAR (`1N = 1000000000000000000000000 yN` or `10^24`).
+- `gas` is how much gas to attach to cover the execution costs. It is in [units of gas](https://docs.near.org/docs/concepts/gas), here 30 [Tera](https://www.nanotech-now.com/metric-prefix-table.htm)Gas—more than enough for this transaction.
 
-- `gas` is how much gas to attach to cover the execution costs. It is in units of gas here `30 Terra gas`, which is more than enough for this transaction.
+You will need to sign in to submit this form. If you don't have a NEAR [testnet](https://docs.near.org/docs/concepts/networks#testnet) account, NEAR Wallet will guide you through creating one.
```

Comment:
```suggestion
You will need to sign in to submit this form. If you don't have a NEAR [testnet](https://docs.near.org/docs/concepts/networks#testnet) account, NEAR Wallet will guide you through creating one and using the one time email verification which makes recovering the account easier.
```

Reply:
Is this worth saying? Have you seen people hesitate to use email recovery on testnet?

Reply:
I just want to direct them to use it vs writing down a seed phrase since they are likely to lose it.